src/main/java/de/fearnixx/jeak/teamspeak/voice/sound/Mp3AudioPlayer.java

```diff
@@ -0,0 +1,213 @@
+package de.fearnixx.jeak.teamspeak.voice.sound;
+
+import com.github.manevolent.ffmpeg4j.*;
+import com.github.manevolent.ffmpeg4j.filter.audio.FFmpegAudioResampleFilter;
+import com.github.manevolent.ffmpeg4j.source.FFmpegAudioSourceSubstream;
+import com.github.manevolent.ffmpeg4j.stream.source.FFmpegSourceStream;
+import de.fearnixx.jeak.teamspeak.voice.sound.opus.OpusParameters;
+import de.fearnixx.jeak.voice.sound.IMp3AudioPlayer;
+
+import java.io.*;
+import java.util.Collection;
+import java.util.concurrent.LinkedBlockingQueue;
+
+public class Mp3AudioPlayer extends TeamspeakFastMixerSink implements IMp3AudioPlayer {
+
+    private static final String MP3_EXTENSION = "mp3";
+
+    static {
+        try {
+            FFmpeg.register();
```

Comment:
Do we need to check for FFmpeg already being registered or is that simply ignored?

Reply:
It seems that is being ignored.